systemtest/exclude_playlist.xml

```diff
@@ -75,4 +74,33 @@
 			<group>system</group>
 		</groups>
 	</test>
+	
+	<!-- Ref: openj9-systemtest/issues/17 -->	
```

Comment:
I believe to disable a testcase we could use the new tag <disabled>. No necessary to move the this file.

Reply:
Yes, you should now be able to use the < disabled / > tag within the playlist.xml file

Reply:
Update made!